### PR TITLE
Add maxHeight option

### PR DESCRIPTION
**Pull request by heatherbooker**
Hi, I've added a maxHeight option to the containerStyle for further convenience, I'd love to see it join the rest of the code in this codebase! Thanks!

### CODE REVIEW

**Comment by tiagovtristao** [comment]
Hi @heatherbooker,

Thanks for the pull request. I think the maxHeight option is very useful indeed and I'm happy to merge it, but what do you think about adding a maxWidth as well?

Thanks!

File: package.json [hunk 1]
```diff
@@ -5,13 +5,13 @@
   "main": "lib/react-table-container.js",
   "module": "es/react-table-container.js",
   "jsnext:main": "es/react-table-container.js",
-  "typings": "./index.d.ts",
+  "typings": "index.d.ts",
```
Comment (tiagovtristao) [inline package.json]: @heatherbooker  What problem did you encounter for this fix?

Comment (heatherbooker) [reply package.json]: I found that npm was not picking up the file; ie, if you npm install react-table-container, then cd into node_modules/react-table/container, index.d.ts is not there.

**Comment by heatherbooker** [comment]
Glad you approve! I agree maxWidth would surely benefit some others but I'm sure minHeight would be useful too and so on, so I'll leave additional options for others to implement - following the steps I took should aid them to that end. Thanks for the review! :)

File: package.json (hunk 1 above)
Comment (tiagovtristao) [reply package.json]: Indeed, it's not copying the file on install. Nice catch! Thanks again :+1:

**Pull request merged**